GDJS/Runtime/events-tools/runtimescenetools.ts

```diff
@@ -281,7 +281,8 @@ namespace gdjs {
         scene: gdjs.RuntimeInstanceContainer,
         externalLayout: string,
         xPos: float,
-        yPos: float
+        yPos: float,
+        zPos: float,
```

Comment:
I feel like I should either set 0 as default value here or deprecate the method to have `createObjectsFromExternalLayout2` because this method is exposed to JS events through `gdjs.evtTools.runtimeScene.createObjectsFromExternalLayout`.
Some JS events might use it and it would create bugs.